### ISSUE TITLE
uMatrix drop-down does not display some sites contacted (out of scope?) - add a field where one can type in an address to create in-context rules on the fly?

### ISSUE THREAD

**Issue by kreil** [gorhill/uMatrix#386]
I love this add-on, and wish to report some issues in the hope that this helps improving it further.

There are a number of sites that will not load correctly even when all fields in the drop-down matrix-menu are green, but do work when filtering for the entire scope has been disabled.

Examples:

uk.businessinsider.com/tony-robbins-how-to-defeat-donald-trump-2015-10
...the video player will only work when filtering for the entire scope has been disabled

https://www.europcar.at/en/fleet
...the pop-up window, when clicking on one of the hyperlink texts of the car class (e.g., "MCMR" next to "Mini) will just display a white blank unless filtering for the entire scope has been disabled.
In this case, I noticed that the log shows an XHR request "https://go.trouter.io/v2/a" that is blocked, but I do not find that domain in the drop down matrix menu.

This page! I could not get the text field to open a new issue on git-hub.

Perhaps there is a better way of fixing that - otherwise how about adding a text field at the bottom of the drop down matrix menu where one can add a domain to create in-scope rules on the fly? Or is the problem an out-of-scope access? In case of the latter, is there any way that uMatrix could help catch and configure these?

Many thanks,
David

**Comment by vzjrz** [comment]
For uk.businessinsider.com/tony-robbins-how-to-defeat-donald-trump-2015-10 it seems that the player needs a domain (ooyala?) that is blacklisted in one of the host files. Setting every column to green will not unblock blacklisted domains in host files you'll have to whitelist them manually.
For https://www.europcar.at/en/fleet I could not get the pop-up window to load at all (would just display a white blank) even on a clean firefox/chrome profile. Didn't see any https://go.trouter.io/v2/a connections either. It might be on their end.
Don't really know what you mean with the github case.

**Comment by seafox** [comment]
No mystery with http://uk.businessinsider.com/tony-robbins-how-to-defeat-donald-trump-2015-10
1. enable these: cf.c.ooyala.com, l.ooyala.com opf.ooyala.com, player.ooyala.com
2. enable script execution cell for code.jquery.com

https://www.europcar.at/en/fleet displays blank pop-ups even on clicking even in IE, so the problem is on the site. Same experience as vzjrz.

**Comment by gorhill** [comment]
Not a uMatrix issue, just a ruleset issue.
